README.md

```diff
@@ -170,3 +171,35 @@ individual rule and how it interprets the incoming packet.
 |------------|------------|---------------------|
 | Index      | VarInt     | 5                   |
 | Enabled    | Boolean    | `false` by default. |
+
+## Sounds
+Noxesium allows servers more control over sounds, when they're started, and while they're playing. 
+You can start a sound from a specific start time, have the sound automatically loop, or modify the
+sound's volume while it's playing.
+
+### Start Sound
+Sounds using this system must be started through the `start_sound` custom packet.\
```

Comment:
I stopped documenting packets in the README.md as it was causing the documentation to be quite long, might need to make a Wiki in the future really. Instead, since packets are fairly clear when looking at the code you can just thoroughly document each field there. You can still note here in the README.md what it does and which packets exist, but you do not need to give a breakdown of the fields. The exact serialization details should be clear from how the client packet is serialized.